cmd/kite/oncall/cmd.go

```diff
@@ -68,13 +69,12 @@ func oncallHandler(cmd *cobra.Command, args []string) (err error) {
 
 	// Fetch oncall data from Platform-SRE team
 	utils.InfoLogger.Print("GET: fetching on-call data of current user team")
-	onCallUsers, err = pdcli.TeamSREOnCall(client)
-
+	onCallLayers, err = pdcli.TeamSREOnCall(client)
 	if err != nil {
 		return err
 	}
 
-	for _, v := range onCallUsers {
+	for _, v := range onCallLayers[2].Users {
```

Comment:
What does `onCallLayers[2]` imply here?

Reply:
When we fetch the list of oncall users and then separate them into layers, there are 5 layers and thus size of `OnCallLayers` is 5
This is because we get the oncall schedule for last 11 hrs and next 13 hrs. This  makes the 2nd layer as the current oncall layer